### PR TITLE
Resolve xgboost benchmark failure

### PR DESCRIPTION
**Pull request by jiaodong**
## Why are these changes needed?

After batch_size in predictor is applied correctly, we can reduce flakiness of our release test by using larger batch size for xgboost, since default 4096 is too small. This reduces runtime from ~310secs to ~200secs.

Full context and debugging history see attached issue.

## Test plan

Run it with new batch size on our release tool

Screen Shot 2022-10-14 at 10 46 57 AM

## Related issue number

#28974

## Checks

- [x] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [x] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by krfricke** [comment]
Can you run the benchmark release test?

File: release/air_tests/air_benchmarks/workloads/xgboost_benchmark.py [hunk 1]
```diff
@@ -102,7 +102,7 @@ def run_xgboost_prediction(model_path: str, data_path: str):
     ds = data.read_parquet(data_path)
     ckpt = XGBoostCheckpoint.from_model(booster=model)
     batch_predictor = BatchPredictor.from_checkpoint(ckpt, XGBoostPredictor)
-    result = batch_predictor.predict(ds.drop_columns(["labels"]))
+    result = batch_predictor.predict(ds.drop_columns(["labels"]), batch_size=8192)
```
Comment (c21) [inline release/air_tests/air_benchmarks/workloads/xgboost_benchmark.py:105]: shall we also add a comment? Thanks.

Comment (jiaodong) [reply release/air_tests/air_benchmarks/workloads/xgboost_benchmark.py:105]: sure i can add one once release test on this PR finish

**Review by amogkam — APPROVED** [review]
LGTM, but lets get a passing release test before merging

**Comment by jiaodong** [comment]
> LGTM, but lets get a passing release test before merging

Somehow this PR's release kick off failed due to missing wheel before test even runs -- is it because this PR didn't trigger a new wheel build ?

**Comment by jiaodong** [comment]
release test https://buildkite.com/ray-project/release-tests-pr/builds/18161#0183d7c9-df5b-4f11-8b32-ba440595d6fe passed

**Review by krfricke — APPROVED** [review]

**Pull request merged**